run_all_trials.bash

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+TEAM_NAME=$1
+
+#TODO: get list of trials automatically
```

Comment:
From `ls comp_configs` minus the extensions. each of the config files in the comp_configs directory is used for a different trial (once).